web-animations-1/Overview.bs

```diff
@@ -822,6 +831,29 @@ in which they are created. Certain <a lt="animation class">classes of
 animations</a>, however, may provide alternative means of ordering animations
 (see [[#animation-classes]]).
 
+To convert a <dfn lt="time-based animation to a proportional animation">,
+time-based animation to a proportional animation</dfn>, effectively the
+[=timeline duration=] is divided up by the relative proportions of the
+specified times of the <a>animation</a>, <var>animation</var>. Concretely:
+
+1.  If the [=iteration duration=] is auto, then perform the following steps.
+
+    1.   Compute [=start delay=] and [=end delay=] to 0, as it is not possible
+         to mix time and proportions. Future versions may allow these properties
+         to be assigned percentages.
+
+1.  Let <var>total time</var> be equal to the result of evaluating
+    <code>|start delay| + |iteration count| * |iteration duration| + |end delay|</code>.
+
+1.  Compute [=start delay=] to be the result of evaluating
+    <code>|start delay| / |total time|</code>.
+
+1.  Compute [=iteration duration=] to be the result of evaluating
+    <code>|iteration duration| / |total time|</code>.
+
+1.  Compute [=end delay=] to be the result of evaluating
+    <code>|end delay| / |total time|</code>.
+
```

Comment:
In this calculation the resulted values sum to 1. However when iteration duration is "auto", the timing values sum to timeline.duration. I think for consistency and correctness the values in both cases should sum to timeline.duration.

Reply:
Good catch, it's possible timeline duration will not always be 100% and we should match that here. Done.